.github/workflows/test.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.6', '1', 'nightly']
+        julia-version: ['1.9', '1', 'nightly']
```

```suggestion
        julia-version: ['1.8', '1', 'nightly']
```

This should work for julia 1.8, no?
We would also need to edit the julia compat entry in Project.toml

---

indeed

---

I don't think we need to change the compat actually since even if const field were introduced in 1.8, all the old syntax still works in 1.6 so I thing narrowing the version is not necessary

---

in that case we should just wrap the offending test in 
```
@static if VERSION >= v"1.8"
...
end
```
and still run tests on 1.6

---

yes this is even better

---

will change that